src/app/app-components/breadcrumbs-navbar/breadcrumbs-navbar.component.html

```diff
@@ -1,4 +1,4 @@
 <div class="rectangle">
     <span><a routerLink="/courses">Courses</a></span>
-    <span *ngIf="courseId">&nbsp;/&nbsp;{{courseTitle}}</span>
+    <span *ngIf="courseId">&nbsp;/&nbsp;{{courseName}}</span>
```

Comment:
We may use CSS paddings to add spacing between HTML elements. For example:

```html
<span>
    <a routerLink="/courses">Courses</a>
</span>

<ng-container *ngIf="courseId">
    <span class='breadcrumb-separator'>/</span>
    <span>{{courseName}}</span>
</ng-container>
```

```css
.breadcrumb-separator{
    padding: 0 .5rem; // or 0 2px;
}
```

Reply:
Done.